Servers/swagger.yaml

```
@@ -3751,8 +3751,158 @@ paths:
+          schema:
+            type: integer
+      responses:
+        "200":
+          description: Task details
+        "401":
+          description: Unauthorized
+        "404":
+          description: Task not found
+        "500":
+          description: Internal server error
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)